concrete/controllers/single_page/dashboard/reports/security.php

```diff
@@ -29,6 +29,7 @@ public function __construct(Page $page, Service $siteService, SiteValidatorManag
     public function view()
     {
         $result = new \ArrayIterator();
+        $this->manager->setEnvironment($this->app->environment());
```

Comment:
Ah, I I just assumed that it would be the site's env. So we need a new page/button that says "THIS IS PRODUCTION MODE"?